File: python vs cpp/4_List python vs  Array cpp/5_array_reverse_print.py
# Input the size of the list
n = int(input())

# Declare a list and input 'n' values
ar = []
for i in range(n):
    ar.append(int(input()))

# Print the list elements in reverse order
for i in range(n - 1, -1, -1):
    print(ar[i], end=" ")

File: 0_python compare to  cpp/4_List python vs  Array cpp/5_array_reverse_print.cpp
#include <iostream>
using namespace std;

int main() {
    int n;
    cin >> n;  // Input the size of the array

    int ar[n];  // Declare an array of size 'n'

    // Input 'n' values into the array
    for (int i = 0; i < n; i++) {
        cin >> ar[i];
    }

    // Print the array elements in reverse order
    for (int i = n - 1; i >= 0; i--) {
        cout << ar[i] << " ";
    }

    return 0;
}
